js/src/ui/TxList/txList.js

```diff
@@ -35,19 +35,16 @@ class TxList extends Component {
       PropTypes.array,
       PropTypes.object
     ]).isRequired,
-    netVersion: PropTypes.string.isRequired
+    blockNumber: PropTypes.object,
+    netVersion: PropTypes.string.isRequired,
+    newError: PropTypes.func
   };
 
-  store = new Store(this.context.api);
-
   componentWillMount () {
+    this.store = new Store(this.context.api, this.props.newError);
```

Comment:
newError won't get loaded in until CWM due to lifetimes. I should really remove:
`this.store.loadTransactions(this.props.hashes);` and have that part of the constructor method too....